### PR TITLE
Open avb next

### PR DESCRIPTION
**Pull request by htfujina**
Man I'm bad at this github stuff.

The whitespace fixes were supposed to be separate from the added devices.

And now my later fixes are in this request as well?

### CODE REVIEW

File: lib/igb/e1000_hw.h [hunk 1]
```diff
@@ -71,11 +71,14 @@ struct e1000_hw;
 #define E1000_DEV_ID_I210_FIBER			0x1536
 #define E1000_DEV_ID_I210_SERDES		0x1537
 #define E1000_DEV_ID_I210_SGMII			0x1538
+#define E1000_DEV_ID_I210_COPPER_FLASHLESS	0x157B
+#define E1000_DEV_ID_I210_SERDES_FLASHLESS	0x157C
```
Comment (andrew-elder) [inline lib/igb/e1000_hw.h]: these #defines are out of order

Comment (andrew-elder) [inline lib/igb/e1000_hw.h]: But I now see they are grouped by device type, so ignore my previous comment.

**Comment by andrew-elder** [comment]
Looks good to me. Can someone else at Intel confirm?

**Comment by eric-mann** [comment]
looks good to me.

**Pull request merged**